Can't run micro 4.1.0

@rauchg Hi, it's me again - It's related to the update to `args@1.2.0` which doesn't have `chalk` as dependency (only devDependency) - Related: https://github.com/leo/args/issues/17

```
⬢  microtest  micro --version
module.js:327
    throw err;
    ^

Error: Cannot find module 'chalk'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/julianduque/.nvm/versions/node/v4.4.5/lib/node_modules/micro/node_modules/args/dist/index.js:33:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
```

@leo not exactly what I'm experiencing right now

```
⬢  ~  npm install micro -g
/Users/julianduque/.nvm/versions/node/v4.4.5/bin/micro -> /Users/julianduque/.nvm/versions/node/v4.4.5/lib/node_modules/micro/build/micro
micro@4.1.0 /Users/julianduque/.nvm/versions/node/v4.4.5/lib/node_modules/micro
├── media-typer@0.3.0
├── babel-plugin-module-alias@1.1.1
├── raw-body@2.1.5 (unpipe@1.0.0, bytes@2.2.0, iconv-lite@0.4.13)
├── args@1.2.0 (camelcase@3.0.0, replaceall@0.1.6, pkginfo@0.4.0, minimist@1.2.0, loud-rejection@1.4.1)
├── micro-core@0.4.0 (babel-regenerator-runtime@6.5.0, isstream@0.1.2, minimist@1.2.0, raw-body@2.1.6)
├── babel-plugin-transform-runtime@6.7.5 (babel-runtime@5.8.38)
├── babel-runtime@6.6.1 (core-js@2.4.0)
├── babel-plugin-transform-async-to-generator@6.4.6 (babel-runtime@5.8.38, babel-plugin-syntax-async-functions@6.8.0, babel-helper-remap-async-to-generator@6.8.0)
├── babel-register@6.5.2 (path-exists@1.0.0, home-or-tmp@1.0.0, mkdirp@0.5.1, source-map-support@0.2.10, babel-runtime@5.8.38, lodash@3.10.1, core-js@1.2.6, babel-core@6.9.1)
└── babel-preset-es2015@6.3.13 (babel-plugin-transform-es2015-literals@6.8.0, babel-plugin-transform-es2015-typeof-symbol@6.8.0, babel-plugin-transform-es2015-arrow-functions@6.8.0, babel-plugin-transform-es2015-template-literals@6.8.0, babel-plugin-transform-es2015-spread@6.8.0, babel-plugin-check-es2015-constants@6.8.0, babel-plugin-transform-es2015-for-of@6.8.0, babel-plugin-transform-es2015-block-scoped-functions@6.8.0, babel-plugin-transform-es2015-function-name@6.9.0, babel-plugin-transform-es2015-destructuring@6.9.0, babel-plugin-transform-es2015-block-scoping@6.9.0, babel-plugin-transform-es2015-parameters@6.9.0, babel-plugin-transform-es2015-classes@6.9.0, babel-plugin-transform-regenerator@6.9.0, babel-plugin-transform-es2015-computed-properties@6.8.0, babel-plugin-transform-es2015-unicode-regex@6.8.0, babel-plugin-transform-es2015-shorthand-properties@6.8.0, babel-plugin-transform-es2015-modules-commonjs@6.8.0, babel-plugin-transform-es2015-sticky-regex@6.8.0, babel-plugin-transform-es2015-object-super@6.8.0)
⬢  ~  micro --version
module.js:327
    throw err;
    ^

Error: Cannot find module 'chalk'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/julianduque/.nvm/versions/node/v4.4.5/lib/node_modules/micro/node_modules/args/dist/index.js:33:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
⬢  ~
```

--- a/bin/micro
+++ b/bin/micro
@@ -6,7 +6,6 @@ import { resolve } from 'path'
 
 // babel
 import preset2015 from 'babel-preset-es2015'
-import syntaxAsync from 'babel-plugin-syntax-async-functions'
 import transformAsync from 'babel-plugin-transform-async-to-generator'
 import transformRuntime from 'babel-plugin-transform-runtime'
 import alias from 'babel-plugin-module-alias'
@@ -38,7 +37,6 @@ if (!flags.noBabel) {
   require('babel-register')({
     presets: [preset2015],
     plugins: [
-      syntaxAsync,
       transformAsync,
       transformRuntime,
       [alias, [

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/zeit/micro",
   "dependencies": {
-    "args": "1.2.0",
+    "args": "1.2.1",
     "babel-plugin-module-alias": "1.1.1",
     "babel-plugin-transform-async-to-generator": "6.4.6",
     "babel-plugin-transform-runtime": "6.7.5",

